src/app/page/cabinet/cabinet.component.ts

```diff
@@ -113,17 +116,28 @@ export class CabinetComponent implements OnInit {
     }
   }
 
-  isOwner(course: CourseLink): boolean {
+  isOwnerCourse(course: CourseLink): boolean {
     return course.ownerId === this.user.id;
   }
 
+  isOwnerDeck(deck: DeckDTO): boolean {
+    return deck.ownerId === this.user.id;
+  }
+
   changeAccess(course: CourseLink, access: boolean) {
     course.published = access;
 
     this.cabinetService.updateCourse(course)
       .subscribe((response) => console.log());
   }
 
+  changeAccessDeck(deck: DeckLinkByCategory, access: boolean) {
+     deck.hidden = access;
+
+    this.cabinetService.toggleDeck(deck)
+      .subscribe((response) => console.log());
```

Comment:
Use the fact
`import {NGXLogger} from 'ngx-logger';`
istead of 
`console.log()`